telco-ran/configuration/kube-compare-reference/metadata.yaml

```diff
@@ -306,6 +306,14 @@ parts:
         allOrNoneOf:
           - path: optional/console-disable/ConsoleOperatorDisable.yaml
 
+  - name: optional-node
+    description: |-
+      Template to enable fetching Node objects
+    components:
+      - name: node-template
+        allOrNoneOf:
+          - path: optional/node/node.yaml
```

Comment:
@dav1x : Did we forget to `git add` this file?

Reply:
I thought I added. Pushed in a new commit.